src/sql-parser/src/ast/defs/expr.rs

```diff
@@ -261,6 +269,31 @@ impl<T: AstInfo> AstDisplay for Expr<T> {
                 f.write_node(&subquery);
                 f.write_str(")");
             }
+            Expr::Like {
+                expr,
+                pattern,
+                escape,
+                case_insensitive,
+                negated,
+            } => {
+                f.write_node(&expr);
+                if *negated {
+                    f.write_str(" NOT");
+                }
+                if *case_insensitive {
+                    f.write_str(" ILIKE ");
+                } else {
+                    f.write_str(" LIKE ");
+                }
+                f.write_node(&pattern);
+                match escape {
+                    Some(x) => {
+                        f.write_str(" ESCAPE ");
+                        f.write_node(x);
+                    }
+                    _ => {}
+                }
```

Comment:
optional, but `if let` is more common in the situation that you only need one arm of the match:

```suggestion
                if let Some(escape) = escape {
                    f.write_str(" ESCAPE ");
                    f.write_node(escape);
                }
```